README.md

```diff
@@ -98,7 +98,7 @@ Add the following to your `Cargo.toml` file:
 
 ```toml
 [dependencies]
-serenity = "0.11"
+serenity = "0.12"
```

Comment:
There's two other Cargo.toml snippets lower down in the README that also need to be updated.